GitUI/CommandsDialogs/FormResetChanges.cs

```diff
@@ -40,6 +40,8 @@ public FormResetChanges(bool hasExistingFiles, bool hasNewFiles)
             {
                 cbDeleteNewFilesAndDirectories.Enabled = true; // A mix of types, so enable the checkbox.
             }
+
+            btnCancel.Select(); // Focus doesn't stick while form is loading/not shown, select does.
```

Comment:
When I see this comment, I wonder if we'd add the `Focus()` call in the 'Shown' event

Reply:
I will check it out in the morning see if.that will work.

Reply:
We have `RuntomeLoaded` (or something like that) that is used to init state when a dialog opens.

Reply:
Also the whole is probably fixable by changing the tabindex of buttons in the designer file.

Reply:
> Also the whole thing is probably fixable by changing the tabindex of buttons in the designer file.

Definitely

Reply:
It was the long way around, but I changed the tabindex & that does let the focus be on cancel when the form appears.